### PR TITLE
[lighthouse] spruce up the homepage

### PR DESCRIPTION
**Pull request by kaycebasques**
* polish all the things
* fixes up https://github.com/GoogleChrome/lighthouse/issues/1842

### CODE REVIEW

File: src/content/en/tools/lighthouse/index.md [hunk 1]
```diff
@@ -70,103 +76,138 @@ failing tests as indicators on what you can do to improve your app.
...
 
-1. Download Google Chrome 54 or later.
-- Install the <a class="gc-analytics-event"
-     data-category="crx-install" data-label="lighthouse-install-button"
-     href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk"
-     title="Install Lighthouse Chrome Extension" target="_blank">
-    <span class="material-icons" style="vertical-align:middle">file_download</span>
-    Lighthouse Chrome Extension</a> from the Chrome Webstore.
+1. Install the <a class="gc-analytics-event" data-category="Lighthouse"
+   data-label="Home / Install Extension" href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk"
```
Comment (ebidel) [inline src/content/en/tools/lighthouse/index.md]: do you want to update the category/labels on the main install button too (one at the top)?

File: src/content/en/tools/lighthouse/index.md [hunk 2]
```diff
@@ -70,103 +76,138 @@ failing tests as indicators on what you can do to improve your app.
...
 
-Note: Lighthouse can also be used programmatically as a Node module. See
-[this example][example] for how call Lighthouse to create custom reports, run
-your own audits, and integrate with Mocha testing.
+#### Run the Node module programmatically {: #programmatic }
+
+See [lighthouse-tests.js][programmatic] for an example of running Lighthouse
+programmatically, as a Node module.
+
+[programmatic]: https://github.com/justinribeiro/lighthouse-mocha-example/blob/master/test/lighthouse-tests.js
```
Comment (ebidel) [inline src/content/en/tools/lighthouse/index.md]: you can link to our readme now!

https://github.com/GoogleChrome/lighthouse#using-programmatically

File: src/content/en/tools/lighthouse/index.md (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/index.md]: Why, yes. Yes, I do.

File: src/content/en/tools/lighthouse/index.md (hunk 2 above)
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/index.md]: good catches thanks @ebidel

File: src/content/en/tools/lighthouse/index.md [hunk 3]
```diff
@@ -70,103 +76,138 @@ failing tests as indicators on what you can do to improve your app.
...
+   version of [Node](https://nodejs.org).
+1. Install Lighthouse. The `-g` flag installs it as a global module.
 
         npm install -g lighthouse
 
-Then, run Lighthouse against a URL to audit the page:
+To run an audit:
 
-    lighthouse https://airhorner.com/
+    lighthouse <url>
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/index.md]: Can I assume a localhost url will work?

File: src/content/en/tools/lighthouse/index.md (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/index.md]: ✔️

File: src/content/en/tools/lighthouse/index.md (hunk 2 above)
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/index.md]: ✔️

File: src/content/en/tools/lighthouse/index.md (hunk 3 above)
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/index.md]: Yup

**Comment by kaycebasques** [comment]
@ebidel I updated per your review LMK if it L👍TY

**Pull request merged**